Dockerfile

```diff
@@ -17,17 +17,17 @@
 # Now you can connect to:
 #    http://172.17.0.1:8891
 #
-FROM ubuntu:14.04
+FROM ubuntu:16.04
 MAINTAINER pietro.delsante_AT_gmail.com
 ENV DEBIAN_FRONTEND noninteractive
 EXPOSE 8891
 
 RUN apt-get update && \
-    apt-get -y install curl && \
+    apt-get -y install curl sudo && \
```

Comment:
Actually I am not sure we need sudo at all, maybe we could safely remove it from the next line as well...